com.unity.render-pipelines.universal/Editor/Camera/URPSerializedCamera.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    internal class URPSerializedCamera
```

Comment:
UniversalRenderPipelineSerializedCamera?

Reply:
Yeah, I will change that too, many thanks for the heads up :)